physical/raft/raft.go

```diff
@@ -477,6 +477,31 @@ func (b *RaftBackend) Close() error {
 	return nil
 }
 
+func (b *RaftBackend) CollectMetrics(sink *metricsutil.ClusterMetricSink) {
+	b.l.RLock()
+	stats := b.stableStore.(*raftboltdb.BoltStore).Stats()
+	b.l.RUnlock()
+	txstats := stats.TxStats
+	sink.SetGauge([]string{"raft", "bolt_store", "freelist", "free_pages"}, float32(stats.FreePageN))
+	sink.SetGauge([]string{"raft", "bolt_store", "freelist", "pending_pages"}, float32(stats.PendingPageN))
+	sink.SetGauge([]string{"raft", "bolt_store", "freelist", "allocated_bytes"}, float32(stats.FreeAlloc))
+	sink.SetGauge([]string{"raft", "bolt_store", "freelist", "used_bytes"}, float32(stats.FreelistInuse))
+	sink.SetGauge([]string{"raft", "bolt_store", "transaction", "started_read_transactions"}, float32(stats.TxN))
+	sink.SetGauge([]string{"raft", "bolt_store", "transaction", "currently_open_read_transactions"}, float32(stats.OpenTxN))
+	sink.SetGauge([]string{"raft", "bolt_store", "page", "count"}, float32(txstats.PageCount))
+	sink.SetGauge([]string{"raft", "bolt_store", "page", "bytes_allocated"}, float32(txstats.PageAlloc))
+	sink.SetGauge([]string{"raft", "bolt_store", "cursor", "count"}, float32(txstats.CursorCount))
+	sink.SetGauge([]string{"raft", "bolt_store", "node", "count"}, float32(txstats.NodeCount))
+	sink.SetGauge([]string{"raft", "bolt_store", "node", "dereferences"}, float32(txstats.NodeDeref))
+	sink.SetGauge([]string{"raft", "bolt_store", "rebalance", "count"}, float32(txstats.Rebalance))
+	sink.AddSample([]string{"raft", "bolt_store", "rebalance", "time"}, float32(txstats.RebalanceTime))
+	sink.SetGauge([]string{"raft", "bolt_store", "split", "count"}, float32(txstats.Split))
+	sink.SetGauge([]string{"raft", "bolt_store", "spill", "count"}, float32(txstats.Spill))
+	sink.AddSample([]string{"raft", "bolt_store", "spill", "time"}, float32(txstats.SpillTime))
+	sink.SetGauge([]string{"raft", "bolt_store", "write", "count"}, float32(txstats.Write))
+	sink.AddSample([]string{"raft", "bolt_store", "write", "time"}, float32(txstats.WriteTime))
```

Comment:
Could we rename these to something like `raft.logstore.bolt....` that way we can add fsm versions too: `raft.fsm.bolt....`

Reply:
Yep, for sure